test/app_profiler/backend_test.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+module AppProfiler
+  class BackendTest < TestCase
+    test ".backend= fails to update the backend if already profiling" do
+      skip("Vernier not supported") unless defined?(AppProfiler::VernierBackend::NAME)
```

Comment:
Let's wrap the entire test case in a ruby version if statement instead.

Reply:
A ruby version if statement is not sufficient on its own, we need to verify if vernier is also supported. I updated it to use the new `vernier_supported?` helper though. This should implicitly ensure that the ruby version is also adequate.